pacman-mirrors/mirrorlist.msys

```diff
@@ -12,3 +12,4 @@ Server = https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/$arch/
 Server = http://mirrors.ustc.edu.cn/msys2/msys/$arch/
 Server = http://mirror.bit.edu.cn/msys2/msys/$arch/
 Server = https://mirror.selfnet.de/msys2/msys/$arch/
+Server = https://mirrors.sjtug.sjtu.edu.cn/msys2/$arch/
```

Comment:
Sorry for the mistake, just now I found that this URL should be /msys2/msys/$arch. I'll make a PR in a day or two.

Reply:
no problem, thanks!